### PR TITLE
MachXO3D fuzzers

### PR DESCRIPTION
**Pull request by mmicko**
Add fuzzers for MachXO3D based on those from MachXO3

### CODE REVIEW

**Comment by mmicko** [comment]
This is now ready to be reviewed, and feature supported are same as on MachXO3

File: fuzzers/machxo3/120-clkdiv/1300/fuzzer.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
```
Comment (gatecat) [inline fuzzers/machxo3/120-clkdiv/1300/fuzzer.py:5]: is this machxo3 change intentional?

File: libtrellis/src/Bitstream.cpp [hunk 2]
```diff
@@ -147,7 +151,7 @@ class BitstreamReadWriter {
 
     // Decompress and copy multiple bytes into an OutputIterator and update CRC
     template<typename T>
-    void get_compressed_bytes(T out, size_t count, array<uint8_t, 8> compression_dict) {
+    void get_compressed_bytes(T out, size_t count, array<uint8_t, 16> compression_dict) {
```
Comment (gatecat) [inline libtrellis/src/Bitstream.cpp:154]: this all looks nicely done, but for piece of mind can you double check a compressed bitstream still works on an ecp5 with these changes?

File: fuzzers/machxo3/120-clkdiv/1300/fuzzer.py (hunk 1 above)
Comment (mmicko) [reply fuzzers/machxo3/120-clkdiv/1300/fuzzer.py:5]: Yes. Parameter was just not used. So it was cleanup

File: libtrellis/src/Bitstream.cpp (hunk 2 above)
Comment (mmicko) [reply libtrellis/src/Bitstream.cpp:154]: Did this while working on PR, but will do it again.

Comment (mmicko) [reply libtrellis/src/Bitstream.cpp:154]: Checked right with couple of bitstreams now on ULX3S board, all fine.

**Comment by mmicko** [comment]
Have resolved issue with conflicting database now, so it is ready for merge

**Pull request merged**